README.md

```diff
@@ -59,6 +59,30 @@ Use `^C` to stop the container and `docker-compose down` to remove the last used
 Many TypeScript and static file changes can be made without re-building the image,
 but as necessary, rebuild the image using Step 1.
 
+#### Step By Step
+
+Explicit steps for setting up CCM in a development environment.
+
+1. `ngrok http 4000`
+1. `curl --silent http://127.0.0.1:4040/api/tunnels | jq -r '.tunnels[0].public_url'`
+   Copy only the hostname, not the `http`/`https` scheme prefix.
+1. `cp config/lti_dev_key_sample.json config/lti_dev_key.json`
+1. `mvim config/lti_dev_key.json`
+   Replace all occurrences of `{ccm_app_url}` with the hostname copied in the earlier step.
+1. Go to Canvas "Developer Keys" management page at (https://umich.instructure.com/accounts/1/developer_keys).  Click the button for "+ Developer Key", then "+ LTI Key".
```

Comment:
It is.  Since the application isn't used by anyone outside U-M yet and it's still very early in the development, I thought that would be acceptable.

I wonder whether there is a way to refer to the developer key settings page without using an organization-specific URL.